<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ventana</class>
 <widget class="QMainWindow" name="Ventana">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos.qrc">
    <normaloff>:/huevo.png</normaloff>:/huevo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pbGeneral">
          <property name="text">
           <string>Agregar Datos Generales de la Sesion</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbRequirente">
          <property name="text">
           <string>Agregar Requirente</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbRequerido">
          <property name="text">
           <string>Agregar Requerido</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Generar Acta de Cierre</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QTextBrowser" name="text"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="pbGuardar">
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionGuardar_Como"/>
    <addaction name="separator"/>
    <addaction name="actionMaximizar"/>
    <addaction name="actionMinimizar"/>
    <addaction name="separator"/>
    <addaction name="actionCerrar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNuevo">
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionGuardar_Como">
   <property name="text">
    <string>Guardar Como</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+G</string>
   </property>
  </action>
  <action name="actionMaximizar">
   <property name="text">
    <string>Maximizar</string>
   </property>
   <property name="shortcut">
    <string>Shift+M</string>
   </property>
  </action>
  <action name="actionMinimizar">
   <property name="text">
    <string>Minimizar</string>
   </property>
   <property name="shortcut">
    <string>Shift+N</string>
   </property>
  </action>
  <action name="actionCerrar">
   <property name="text">
    <string>Cerrar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pbGeneral</tabstop>
  <tabstop>pbRequerido</tabstop>
  <tabstop>pbGuardar</tabstop>
 </tabstops>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
